src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_EI_ApprenticeshipDaysInLearning.sql

```diff
@@ -1,10 +1,17 @@
 ﻿CREATE TABLE [ASData_PL].[EI_ApprenticeshipDaysInLearning]
 (
-	[LearnerId]							[uniqueidentifier]	NOT NULL PRIMARY KEY,
+	[LearnerId]							[uniqueidentifier]	NOT NULL,
 	[NumberOfDaysInLearning]			[int]				NULL,
-	[CollectionPeriodNumber]			[tinyint]			NULL,
-	[CollectionPeriodYear]				[smallint]			NULL,
+	[CollectionPeriodNumber]			[tinyint]			NOT NULL,
+	[CollectionPeriodYear]				[smallint]			NOT NULL,
 	[CreatedDate]						[datetime2](7)		NULL,
 	[UpdatedDate]						[datetime2](7)		NULL,
-	[AsDm_UpdatedDateTime]				[datetime2](7)		DEFAULT (getdate())
-) 
\ No newline at end of file
+	[AsDm_UpdatedDateTime]				[datetime2](7)		DEFAULT (getdate()),
+	PRIMARY KEY CLUSTERED 
+	(
+		[LearnerId] ASC,
+		[CollectionPeriodNumber] ASC,
+		[CollectionPeriodYear] ASC
+	)
+) 
```

Comment:
We might have more than one submission on the Learner. So, It might not always be unique.

Reply:
Taken off primary key all together